packages/escape-html/src/index.js

```diff
@@ -26,12 +26,14 @@ const REGEXP_INVALID_ATTRIBUTE_NAME = /[\u007F-\u009F "'>/="\uFDD0-\uFDEF]/;
  * @see https://w3c.github.io/html/syntax.html#ambiguous-ampersand
  * @see https://w3c.github.io/html/syntax.html#named-character-references
  *
- * @param {string} value Original string.
+ * @param {string}  value Original string.
+ * @param {boolean} all   Set to true to escape all ampersands.
```

Comment:
this feels a bit mystical. is there any way we can guide people on when they should set `all`? 

what reasons would we choose one or the other? is `all` ever appropriate?

Reply:
Yeah, this can be better documented. Escaping all ampersands is appropriate in the context of editing the HTML. This is the key to fixing the problem. I think I'll split this in two separate functions and document the new one better.